README.md

```diff
@@ -665,9 +799,258 @@ Operands:
+
+Errors: 
+- `BadOrigin`
+- `ReanchorFailed`
+- `AssetNotFound`
+- `Unroutable`
+- `DestinationUnsupported`
+- `ExceedsMaxMessageSize`
+- `Transport`
+- `NotHoldingFees`
+- `FailedToTransactAsset`
+- `LockError`
+
+### [`UnlockAsset`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L935)
+
+Remove the lock over `asset` on this chain and (if nothing else is preventing it) allow the asset to be transferred.
+
+Operands:
+- `asset: MultiAsset`: The asset to be unlocked.
+- `target: MultiLocation`: The owner of the asset on the local chain.
+
+Kind: *Instruction*
+
+Errors: 
+- `BadOrigin`
+- `AssetNotFound`
+- `LockError`
+
+### [`NoteUnlockable`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L950)
+
+Asset (`asset`) has been locked on the `origin` system and may not be transferred. It may only be unlocked with the receipt of the [`UnlockAsset`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L935) instruction from this chain.
+
+Operands:
+
+- `asset: MultiAsset`: The asset(s) which are now unlockable from this origin.
+- `owner: MultiLocation`: The owner of the asset on the chain in which it was locked. This may be a location specific to the origin network.
+
+Safety: `origin` must be trusted to have locked the corresponding `asset` prior as a consequence of sending this message.
+
+Kind: *Trusted Indication*
+
+Errors: 
+- `BadOrigin`
+- `AssetNotFound`
+- `LockError`
+
+### [`RequestUnlock`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L964)
+
+Send an [`UnlockAsset`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L935) instruction to the `locker` for the given `asset`. This may fail if the local system is making use of the fact that the asset is locked or, of course, if there is no record that the asset actually is locked.
+
+Operands:
+- `asset`: The asset(s) to be unlocked.
+- `locker`: The location from which a previous [`NoteUnlockable`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L950) was sent and to which an [`UnlockAsset`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L935) should be sent.
+
+Kind: *Instruction*
+
+Errors: 
+- `BadOrigin`
+- `ReanchorFailed`
+- `AssetNotFound`
+- `Unroutable`
+- `DestinationUnsupported`
+- `ExceedsMaxMessageSize`
+- `Transport`
+- `NotHoldingFees`
+- `FailedToTransactAsset`
+- `LockError`
+
+### [`SetFeesMode`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L974)
+
+Sets the Fees Mode Register.
+	
+- `jit_withdraw: bool`: The fees mode item; if set to `true` then fees for any instructions are withdrawn as needed using the same mechanism as `WithdrawAssets`.
```

Comment:
This is not a good explanation, because naturally people will ask why we don't simply just use `WithdrawAsset` to do the job. And I don't believe it is true that _any_ fees are eligible to be withdrawn -- only a select few instructions do this.

Reply:
This better?